### DISCUSSION TITLE
Hoymiles HMS-1800 or 2000 throttled down?

### DISCUSSION THREAD

**Discussion by greymda** [hoylabs/OpenDTU-OnBattery#1429] [Q&A]
hey guys. if i buy a Hoymiles 1800 or 2000 will it be throttled down to 800W or i can (somehow) use all its power, say 2kWh?

**Comment by Sorbat** [comment]
> hey guys. if i buy a Hoymiles 1800 or 2000 will it be throttled down to 800W or i can (somehow) use all its power, say 2kWh?

can you explain this in more detail?
